docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,107 @@
+
+[discrete]
+[[macos-permissions-ven]]
+== macOS permissions
+
+The behavior of the Endgame sensor differs based on your macOS version. MDM/Jamf users can pre-approve all Full Disk Access without granting permission to the sensor. However, depending on the macOS version and sensor type, non-MDM/Jamf users may be prompted to enable Full Disk Access for required security files.
+
+Here are the following Full Disk Access requirements for specific versions of macOS:
+
+- `10.15`: You cannot install the sensor without allowing it to load a <<kernel-ext-approval-endgame, kernel extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, installation proceeds.
+
+- `11.0`, `12.0`: You cannot install the sensor without allowing it to load a <<system-extension, system extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, a second prompt appears to enable Network Filtering. Approve this final prompt to proceed.
++
+You must also grant Full Disk Access to `com.endgame.systemextension`.
+
+- `10.15, 11.0, 12.0`: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Endgame sensor only. To see requirements for the {elastic-endpoint}, refer to <<deploy-elastic-endpoint>>.
+
+[discrete]
+[[kernel-ext-approval-endgame-ven]]
+== Approve the kernel extension for the Endgame sensor
+
+For endpoints running macOS Mojave (10.14) and earlier, the Endgame sensor will attempt to load a kernel extension (as opposed to a system extension) during installation. This kernel extension is required to provide insight into system events such as process events, file system events, and network events. The following prompt appears during installation:
+
+--
+image::images/fda/sys-ext-blocked.png[System extension blocked]
+--
+
+To approve the extension:
+
+. Click *Open Security Preferences*.
+. In the lower-left corner of the pane, click the **Lock button**, then enter your credentials to authenticate.
+. Click *Allow* to load the kernel extension.
++
+--
+image::images/fda/allow-kernel-ext.png[Allow kernel extension]
+--
++
+
+If the prompt does not appear, enable the extension by doing the following:
+
+. Open a Terminal application.
+. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary.
+You should receive an output similar to `149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`. If you receive this output, the kernel extension is enabled.
```

Comment:
It'll be easier to read if this is on a separate line:
`149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`
Same comment for the corresponding line in `endgame-sensor-FDA.asciidoc`

Reply:
Made this change in both places in [ca7dc58](https://github.com/elastic/security-docs/pull/3565/commits/ca7dc5896f91b6541a9c2bdaa8c6b012b531795e).